setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
     invenio-records-resources>=6.0.0,<7.0.0
     invenio-github>=1.0.0,<2.0.0
     invenio-i18n>=2.0.0,<3.0.0
-    invenio-jobs>=1.0.0,<2.0.0
+    invenio-jobs>=2.0.0,<3.0.0
```

Comment:
please warn Zenodo about this bump, just in case
do we need to change anything in https://github.com/CERNDocumentServer/cds-rdm/blob/master/site/cds_rdm/jobs.py or https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/jobs.py (or any other jobs) to adapt to the major bump? ping @ntarocco @slint

Reply:
I think that @anikachurilova had a look already? Can you please confirm?